server/server.go

```diff
@@ -236,6 +240,8 @@ func (r *rodeServer) initialize(ctx context.Context) error {
 			return err
 		}
 	}
+	// Create an index for policy storage
+	r.esClient.Indices.Create(rodeElasticsearchPoliciesIndex)
```

Comment:
We may want some error handling here, but that could be in the follow up

Reply:
I was about to do that, but i couldn't for the life of me get an error.
If an index already exists, this is just successful.